api-report/telemetry-utils.api.md

```diff
@@ -23,9 +23,13 @@ import { TypedEventEmitter } from '@fluidframework/common-utils';
 
 // @public
 export class ChildLogger extends TelemetryLogger {
+    static create(baseLogger?: ITelemetryBaseLogger, namespace?: string, properties?: ITelemetryLoggerPropertyBags, globalProperties?: ITelemetryLoggerPropertyBags, layerVersion?: string): TelemetryLogger;
     // (undocumented)
-    protected readonly baseLogger: ITelemetryBaseLogger;
-    static create(baseLogger?: ITelemetryBaseLogger, namespace?: string, properties?: ITelemetryLoggerPropertyBags): TelemetryLogger;
+    protected get globalProperties(): Required<ITelemetryLoggerPropertyBags>;
+    // Warning: (ae-forgotten-export) The symbol "ChildLoggerRoot" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    protected readonly rootLogger: ChildLoggerRoot;
```

Comment:
This seems like limiting logger nesting / dependency to always be straight line (based on glancing at PR - up to a point where it's not / where parent does not satisfy some signature). For example, this would not work for MultiSinkLogger, right?